src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: 'Roboto font', sans-serif;
```

Comment:
The font-family declaration 'Roboto font' seems incorrect. It should be 'Roboto', sans-serif; ensure that the font name matches the one provided by Google Fonts or any other font service you are using.